inst/templates/pull_request_template.md

```diff
@@ -1,21 +1,94 @@
+    - [ ] Lines are at most 100 characters long
+    - [ ] Consistent use of `<-` not `=` for assignment
+    - [ ] Object names include only alphanumeric characters and underscores (no dots)
+    - [ ] Object names are meaningful, descriptive, and unique (avoid overwriting previous variable)
+    - [ ] Rmd code chunk names are descriptive and use dashes (not underscores or spaces)
+    - [ ] File paths are relative (except for trials and network drive paths) and portable across operating systems (use `file.path()`)
+    - [ ] Functions are organized and well-documented, with explanations of purpose, inputs, and ouput
+    - [ ] Hard coding and magic numbers are avoided
+    - [ ] Comments are helpful and do not include debt (e.g. `# TODO:`) or commented-out backup code
+
+### Peer writing review (use for PRs with PDF and/or Word drafts of PT reports)
+
+See also [writing review guidelines](https://github.com/FredHutch/VISC-Documentation/tree/main/Writing_Reviewing/writing_reviewing_guidelines.md)
 
-## Checklist
+- [ ] The latest version of the report has been compiled to both PDF and Word without errors
+    - [ ] I've opened and reviewed the compiled PDF document
+    - [ ] I've opened and reviewed the compiled Word document
+- [ ] There are no obvious Markdown/pandoc/Latex errors 
+    - [ ] No broken references (?? or ???) in the text (Use find: “??”)
+    - [ ] No blank pages 
+    - [ ] Page x out of y is correct (sometimes y is wrong)  
+- [ ] The reproducibility tables look correct
+    - [ ] The reproducibility tables do not include `NA`, local installations, or unnescessary packages
+    - [ ] The most recent versions (note: not the development versions) of VISCtemplates and VISCfunctions are used
+- [ ] The report text, including figure and table captions, follows VISC conventions (refer to the [writing guidelines](https://github.com/FredHutch/VISC-Documentation/tree/main/Writing_Reviewing) as needed)
+    - [ ] The sample type is accurate (e.g., serum, plasma, PBMC)
+    - [ ] Objectives follow the SAP and study protocol
+    - [ ] Results and summary of main results sections map to the objectives
+    - [ ] Text is spell-checked (including captions and footnotes)
+    - [ ] The correct tense (generally past tense) is used throughout the report
+    - [ ] Capitalization is correct and consistent
+    - [ ] Acronyms and abbreviations are introduced the first time they are used
+- [ ] I have reviewed the results carefully
+    - [ ] Everything mentioned in the Summary of Main Results is also in the Results section
+    - [ ] Statements in Results section are correct (including p-values) and supported by the correct figure and table references
+        - [ ] Variables are used in referring to numeric values in the Results section (to minimize human error)
```

Comment:
I'd prefer "inline referencing" or something similar sentiment (code-based?). I often use filtering commands and do not create variables